website/docs/tutorials/nextjs.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Setup Lingui in Next.js
+description: Learn how to setup and use Lingui with Next.js
+---
+
+### Installation
+
+After creating your Next.js project install the Lingui dependencies. Make sure to install a version greater or equal to [4.8.0](https://github.com/lingui/js-lingui/releases/tag/v4.8.0). 
+
+```
+npm i @lingui/core @lingui/react
+```
+
+```
+npm i -D @lingui/cli @lingui/loader @lingui/macro @lingui/swc-plugin
+```
+
+### Config
+
+In the `next.config.js` file we can add 2 configurations to make our life a little easier. The `swc-plugin` to be able to use [Lingui macros](https://lingui.dev/ref/macro) and the [`@lingui/loader`](https://lingui.dev/ref/loader) to load translation files without compiling them to js first. Both steps are optional if you for example are not using macros.
+
+```js
+//next.config.js
+/** @type {import('next').NextConfig} */
+module.exports = {
+  // to use lingui macros
+  experimental: {
+    swcPlugins: [
+      ['@lingui/swc-plugin', {}],
+    ],
+  },
+  // To import .po (or other formats) translation files directly  
+  webpack: (config, options) => {
+    config.module.rules.push({
+      test: /\.po/,
+      use: [
+        options.defaultLoaders.babel,
```

Comment:
do we need this line? @lingui/loader should return a ready to use module for webpack